firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidator.java

```diff
@@ -134,6 +143,10 @@ private boolean isValidTrace(@Nullable TraceMetric trace, int deep) {
     if (!hasValidAttributes(trace.getCustomAttributesMap())) {
       return false;
     }
+    if (isScreenTrace(trace) && !isValidScreenTrace(trace)) {
+      logger.warn("non-positive totalFrames in screen trace " + trace.getName());
+      return false;
+    }
```

Comment:
Nit: Can we bump this check to line 138. Basically the validation in the increasing order of importance and frequency of occurrence for these checks.

Reply:
done